src/main/java/org/terasology/logic/health/HealthCommands.java

```diff
@@ -72,6 +74,73 @@ public String damageWithType(@Sender EntityRef client, @CommandParam("damageType
             return "Specified damage type does not exist.";
         }
     }
+    public void setAll(DamageResistComponent damageResist,int v){
+        damageResist.damageTypes=new HashMap<>(6);
+        damageResist.damageTypes.put("directDamage",v);
```

Comment:
Somewhat akin to what Jelly already said, but a slightly different angle and still maybe not for this PR: rather than in-lining the values here it may be wise to introduce a set of common damage types constants somewhere (if we don't already have those?) and use those instead here. Still as plain Strings, not like references or enums or anything, but just to avoid defining the values in multiple places.